pkg/gh/github.go

```diff
@@ -75,7 +75,7 @@ func (o GitOptions) handleRateLimit(action func() (*github.Response, error)) err
 		if githubErr := github.CheckResponse(resp.Response); githubErr != nil {
 			isRateLimited, delay := o.checkRateLimiting(githubErr)
 			if isRateLimited {
-				o.Logger.Printf("retrying again later with %d second delay due to secondary rate limiting.", delay)
+				o.Logger.Printf("retrying again later with %d second delay due to secondary rate limiting.", delay.Seconds())
```

Comment:
%f?

Reply:
ah yeah - just pushed a commit to fix that